charts/weblate/values.yaml

```diff
@@ -147,27 +147,29 @@ tolerations: []
 affinity: {}
 
 postgresql:
-  # postgresql.postgresqlUsername -- PostgreSQL user should be a superuser to
+  auth:
+  # PostgreSQL user should be a superuser to
   # be able to install pg_trgm extension. Alternatively you can install it
   # manually prior starting Weblate.
```

Comment:
This comment is no longer visible in the documentation (also it makes little sense now when the username configuration is gone, see other comment).

Reply:
I left it because i felt that if using an external database it's good to know that weblate need an admin user and why. But I can remove that if wanted